pcsx2/PINE.cpp

```diff
@@ -34,23 +32,23 @@
 #define close_portable(a) (close(a))
 #include <sys/socket.h>
 #include <sys/un.h>
+#include <unistd.h>
 #endif
 
 #include "Common.h"
 #include "Memory.h"
-#include "gui/AppSaveStates.h"
-#include "gui/AppCoreThread.h"
-#include "gui/SysThreads.h"
+#include "Elfheader.h"
+#include "pcsx2/VMManager.h"
 #include "svnrev.h"
 #include "PINE.h"
 
-PINEServer::PINEServer(SysCoreThread* vm, unsigned int slot)
-	: pxThread("PINE_Server")
+PINEServer::PINEServer(int slot)
```

Comment:
Use initializer list, i.e.
```suggestion
PINEServer::PINEServer(int slot)
   : m_slot(slot)
```